src/ulong_extras.h

```diff
@@ -77,8 +77,14 @@ ulong n_flog(ulong n, ulong b);
 ulong n_clog(ulong n, ulong b);
 ulong n_clog_2exp(ulong n, ulong b);
 
-ulong __gmpn_gcd_11(ulong, ulong);
-ulong __gmpn_gcd_1(nn_srcptr, long int, ulong);
+#ifdef _MSC_VER
+# define DECLSPEC_IMPORT __declspec(dllimport)
+#else
+# define DECLSPEC_IMPORT
+#endif
+DECLSPEC_IMPORT ulong __gmpn_gcd_11(ulong, ulong);
+DECLSPEC_IMPORT ulong __gmpn_gcd_1(nn_srcptr, long int, ulong);
```

Comment:
Should the second param really be ``long int`` here, not ``slong``?

Reply:
Yes, unless you are on a Cray supercomputer, `mp_size_t` is defined as `long int`.